### ISSUE TITLE
Add saturated colors mode

### ISSUE THREAD

**Issue by Sebash93** [ericmatte/ad-media-lights-sync#10]
**Problem**
I've notice that most of the colors generated by the palette are not being well represented by the LEDS. It seems that when a color does not have enough saturation it is represented by white or blue which is very distant to a representative color of song's cover.

**Solution**
One solution i found was to calculate a saturated version of that color. I did that by converting RGB to HLS and setting Hue with 100% saturation and 50% lightness using the colorsys library.

That turned to be way more effective than the method without conversion (Which 50% of the time was on white or blue). This method is not 100% accurate since you may get some different HUEs  from song's cover, but that happens on the ~20% of the time.

On the other hand i'm very new on writing Python so i'm not aware of code styling or easier ways to achieve the same thing, but i'll leave my  here code in case someone is interested in following this up.

I added colorsys package

`import colorsys`

Then modified set_light_rgb method:
```py
def set_light_rgb(self, light, color):
    """Change a light color."""
    hls = colorsys.rgb_to_hls(color[0]/255, color[1]/255, color[2]/255)
    hls_saturated = [hls[0], 0.5, 1]
    rgb_saturated = colorsys.hls_to_rgb(*hls_saturated)
    rgb_color = [int(rgb_modified[0] * 255), int(rgb_modified[1] * 255), int(rgb_modified[2] * 255)]
    light_kwargs = { "rgb_color": rgb_color }
    if not self.use_current_brightness:
        light_kwargs["brightness"] = 255
    self.turn_on(light, **light_kwargs)
````

**Comment by ericmatte** [comment]
Hi @Sebash93,

Thank you for your suggestion, I will look into it !

However, I find this issue a little odd, because from all my tests, the colors generated on my setup are pretty accurate to the main colors of the current song's cover. They seems to have already have a lot of saturation 🤔 

If you have some songs that I could use to reproduce your issue, that could help.

<details>
<summary>Here's an example on my setup:</summary>
<br>

![image](https://user-images.githubusercontent.com/9013072/88119895-b5dad200-cb8f-11ea-8ff8-c210bc28e3fa.png)

</details>

**Comment by ericmatte** [comment]
Also, in your modified `set_light_rgb` method, `rgb_modified` doesn't seem to be defined.

**Comment by ericmatte** [comment]
Hi @Sebash93,

Sorry for the time it took for me to address this issue. I've added a new option `use_saturated_colors` that uses the code you've suggested for increased saturation.

It is available in the [**v0.4.0**](https://github.com/ericmatte/ad-media-lights-sync/releases/tag/v0.4.0) release.

The only difference is that I use 50% brightness and 50% saturation instead of 100% saturation. In my test, using 50% saturation is still a huge difference when compared to the original color, but also allow more color diversity/accuracy in my opinion.

Let me know what you think of it.

Cheers!
